.drone.star

```diff
@@ -1430,9 +1430,9 @@ def buildOcisWeb():
 		'pull': 'always',
 		'commands': [
 			'cd $GOPATH/src/github.com/owncloud/ocis',
-			'cd ocis-phoenix',
+			'cd web',
 			'make build',
-			'cp bin/ocis-phoenix /var/www/owncloud'
+			'cp bin/web /var/www/owncloud/ocis-web'
```

Comment:
Actually I  wonder why this step exists at all - from `owncloud/web` we test with a version of `web` that is built from the PR branch in `owncloud/web`. We should no need the version that is currently in `owncloud/ocis` `web` folder.

But I can try removing this step completely in a followup PR. Because this looks like it was not needed even when it was called `ocis-phoenix`

Reply:
Note: there is already the "real" `/var/www/owncloud/web` that has been built from the `owncloud/web` branch, which is what we use for testing. So this copy has to go somewhere different, otherwise it gives an error message if we try to copy it on top of the existing `/var/www/owncloud/web`